framework/src/modules/chain/bft/finality_manager.js

```diff
@@ -200,20 +204,20 @@ class FinalityManager {
 			return false;
 		}
 
-		const higherPairVoted = Object.entries(this.preVotes)
+		const highestHeightPreVoted = Object.keys(this.preVotes)
 			.reverse()
-			.find(pair => pair[1] >= this.preVoteThreshold);
+			.find(key => this.preVotes[key] >= this.preVoteThreshold);
 
-		this.prevotedConfirmedHeight = higherPairVoted
-			? parseInt(higherPairVoted[0], 10)
+		this.prevotedConfirmedHeight = highestHeightPreVoted
+			? parseInt(highestHeightPreVoted, 10)
```

Comment:
Do we need to specify the base?

Reply:
Some es-lint rule introduced, which required to specify it.